From 223d67839f1bbb8a0489908bd8f90c63589d80b9 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 14 Nov 2013 14:29:51 +0800
Subject: [PATCH 1056/1072] ENGR00287992-2 usb: doc: chipidea: imx: add
 imx6-usb-charger-detection property

git://git.freescale.com/imx/linux-2.6-imx.git imx_3.10.17_1.0.0_beta
commit dd44146e9d591fff3a6b7f0c2ddff3c37fab8c36

It is used to indicate whether we use SoC's usb charger
detection or not. Besides, we add anatop phandle since
we need to use anatop register to do most of charger detect operations.

Signed-off-by: Peter Chen <peter.chen@freescale.com>
Signed-off-by: Hongbo Zhong <hongbo.zhong@windriver.com>
---
 .../devicetree/bindings/usb/ci13xxx-imx.txt        |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/Documentation/devicetree/bindings/usb/ci13xxx-imx.txt b/Documentation/devicetree/bindings/usb/ci13xxx-imx.txt
index b4b5b79..226ca06 100644
--- a/Documentation/devicetree/bindings/usb/ci13xxx-imx.txt
+++ b/Documentation/devicetree/bindings/usb/ci13xxx-imx.txt
@@ -18,6 +18,12 @@ Optional properties:
 - vbus-supply: regulator for vbus
 - disable-over-current: disable over current detect
 - external-vbus-divider: enables off-chip resistor divider for Vbus
+- imx6-usb-charger-detection: enable imx6 usb charger detect function,
+only set it when the user wants SoC usb charger detection capabilities.
+If the user wants to use charger IC's usb charger detection capabilities,
+please do not set it.
+- fsl,anatop: phandle for anatop module, anatop module is only existed
+at imx6 SoC series
 
 Examples:
 usb@02184000 { /* USB OTG */
@@ -28,4 +34,6 @@ usb@02184000 { /* USB OTG */
 	fsl,usbmisc = <&usbmisc 0>;
 	disable-over-current;
 	external-vbus-divider;
+	imx6-usb-charger-detection;
+	fsl,anatop = <&anatop>;
 };
-- 
1.7.5.4

